b2/console_tool.py

```diff
@@ -355,6 +355,100 @@ def run(self, args):
         return 0
 
 
+class CopyFile(Command):
+    """
+    b2 copy-file [--metadataDirective [copy|replace]] [--contentType <contentType>] \\
+                 [--info <key>=<value>]* [--range start,end] \\
+                 <sourceFileId> <destinationBucketName> <b2FileName>
+
+        Copy a file to the given bucket. Uploads the contents
+        of the source B2 file to destination bucket,
+        and assigns the given name to the new B2 file.
+
+        By default, it copies the file info and content type but you can replace
+        by setting the metadataDirective to 'replace'.
+
+        Content type is optional.  If not set, it will be set based on the
+        source file. It should only be provided when metadataDirective is replace and
+        should not be provided when metadataDirective is copy.
+
+        By default, the whole file gets copied but you can copy a range of bytes
+        from the source file to the new file.
```

Comment:
Need to say whether the range is inclusive or exclusive.

Reply:
I think you might have reviewed an old version of this PR, the current wording is `By default, the whole file gets copied but you can copy an (inclusive!) range of bytes from the source file to the new file.`